test-data/unit/check-expressions.test

```diff
@@ -952,6 +952,16 @@ y: Gen[Literal[1]] = assert_type(Gen(1), Gen[Literal[1]])
 
 [builtins fixtures/tuple.pyi]
 
+[case testAssertTypeUncheckedFunction]
+from typing_extensions import assert_type, Literal
+def f():
```

Comment:
I think that this can be affected by https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-check-untyped-defs 

Please, add a second test with this option on.

Could be:

```python
[case testAssertTypeUncheckedFunctionWithUntypedCheck]
# flags: --check-untyped-defs
...
```